js/data/odata/store.js

```diff
@@ -84,6 +84,10 @@ var ODataStore = Store.inherit({
          * @type boolean
          * @default false
          */
+        /**
+         * @name ODataStoreOptions.stringToLower
```

Comment:
Since `ODataContext` includes the `SharedMethods` mixin as well as `ODataContext` the `@name ODataContext.stringToLower` should be documented and tested too. Contact me for more details.